src/mlpack/methods/ann/activation_functions/hyper_sinh_function.hpp

```diff
@@ -0,0 +1,95 @@
+/**
+ * @file methods/ann/activation_functions/hyper_sinh_function.hpp
+ * @author Mayank Raj
+ *
+ * Definition and implementation of the Hyper-sinh function.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#ifndef MLPACK_METHODS_ANN_ACTIVATION_FUNCTIONS_HYPER_SINH_FUNCTION_HPP
+#define MLPACK_METHODS_ANN_ACTIVATION_FUNCTIONS_HYPER_SINH_FUNCTION_HPP
+
+#include <mlpack/prereqs.hpp>
+
+namespace mlpack {
+
+/**
+ * The Hyper-sinh function, defined by
+ *
+ * @f{eqnarray*}{
+ * f(x) &=& \begin{cases}
+ * \sinh(x) / 3, & \text{if } x > 0 \\
+ * (x^3) / 4, & \text{if } x \leq 0
+ * \end{cases} \\
+ * f'(x) &=& \begin{cases}
+ * \cosh(x) / 3, & \text{if } x > 0 \\
+ * (3/4) x^2, & \text{if } x \leq 0
+ * \end{cases} \\
+ * @f}
+ */
+class HyperSinhFunction
+{
+ public:
+  /**
+   * Computes the Hyper-sinh function.
+   *
+   * @param x Input data.
+   * @return f(x).
+   */
+  static double Fn(const double x)
+  {
+    if (x > 0)
+      return (std::sinh(x) / 3.0);
+    else
+      return (std::pow(x, 3.0   ) / 4.0);
```

Comment:
```suggestion
      return (std::pow(x, 3.0) / 4.0);
```

Remove the extra space.